torchvision/ops/deform_conv.py

```diff
@@ -179,14 +179,16 @@ def forward(self, input: Tensor, offset: Tensor, mask: Optional[Tensor] = None)
         )
 
     def __repr__(self) -> str:
-        s = self.__class__.__name__ + "("
-        s += "{in_channels}"
-        s += ", {out_channels}"
-        s += ", kernel_size={kernel_size}"
-        s += ", stride={stride}"
-        s += ", padding={padding}" if self.padding != (0, 0) else ""
-        s += ", dilation={dilation}" if self.dilation != (1, 1) else ""
-        s += ", groups={groups}" if self.groups != 1 else ""
-        s += ", bias=False" if self.bias is None else ""
-        s += ")"
-        return s.format(**self.__dict__)
+        s = (
+            f"{self.__class__.__name__}("
+            f"{self.in_channels}"
+            f", {self.out_channels}"
+            f", kernel_size={self.kernel_size}"
+            f", stride={self.stride}"
+            f"{f', padding={self.padding}' if self.padding != (0, 0) else ''}"
+            f"{f', dilation={self.dilation}' if self.dilation != (1, 1) else ''}"
+            f"{f', groups={self.groups}'  if self.groups != 1 else ''}"
+            f"{', bias=False' if self.bias is None else ''}"
```

Comment:
Did you miss a nested `f`?
```suggestion
            f"{f', bias=False' if self.bias is None else ''}"
```

Reply:
in this case there is no need because 'bias=False is just a string without placeholders

Reply:
Oh yes, you are right. Let's go with @NicolasHug's proposal below to avoid nesting and make the code easier to understand.

Reply:
missed this, but I already removed the nested fstrings (different from @NicolasHug  proposal). Despite his proposal is more in line with the spirit of this PR I think we can merge as it is if you agree.